src/main/java/com/training/utilities/JsonReaderUtility.java

```diff
@@ -0,0 +1,25 @@
+package com.training.utilities;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+public class JsonReaderUtility {
+    public JsonObject jsonObject;
+
+    public String getJson(String key) {
```

Comment:
I think this method is a lot expensive, as everytime you call `getJson()` it is going to read that file and parse it. So to avoid it lets make this call only once in our life cycle by making this return a hashmap which will have all the key/values of the json file.
Then you just need to call this function in your web base class and store the hashmap in a variable. You can then use this variable in your tests.